src/components/PatientCreationPage/TriagePills.tsx

```diff
@@ -0,0 +1,210 @@
+import React from "react";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import { triageLevel } from "../../graphql/queries/templates/patients";
+import { Colours } from "../../styles/Constants";
+
+const TriagePills = ({
+  currentStatus,
+  handleChange,
+  error,
+  helperText,
+}: {
+  currentStatus: triageLevel | null;
+  handleChange: (
+    e: React.MouseEvent<HTMLElement>,
+    newTriage: triageLevel
+  ) => any;
+  error?: boolean;
+  helperText?: string;
+}) => {
+  const classes = useCompletePatientButtonStyles();
+  return (
+    <Container className={classes.root}>
+      <Typography className={classes.label}>Triage:</Typography>
+
+      <ToggleButtonGroup
+        value={currentStatus}
+        exclusive
+        onChange={handleChange}
+        aria-label="text alignment"
+        classes={{
+          groupedHorizontal: classes.buttonGroup,
+        }}
+      >
+        <ToggleButton
+          value={triageLevel.RED}
+          classes={{
+            root: classes.redpill,
+            selected: classes.selectedRedPill,
+          }}
+        >
+          <Typography variant="body2">Red</Typography>
+        </ToggleButton>
```

Comment:
this bit can be refactored into a loop - if you have a lot of repeated things, consider pulling out the unique stuff and looping the rest of it both for consistency (all your buttons will be the same button) and readability